tests/integration/security/util/connection/checker.go

```diff
@@ -52,7 +53,7 @@ func (c *Checker) Check() error {
 }
 
 func (c *Checker) CheckOrFail(t test.Failer) {
-	if err := retry.UntilSuccess(c.Check); err != nil {
+	if err := retry.UntilSuccess(c.Check, retry.Delay(time.Millisecond*100)); err != nil {
```

Comment:
If we end up failing the request we will tr to call every 10ms which may overload the service. Backoff a bit more aggressively here to be safe. We have had issues where we use up all ephemeral ports as well, so may help avoid those types of issues